packages/teams-js/src/internal/communication.ts

```diff
@@ -1090,6 +1098,7 @@ function createMessageRequest(
     uuid: messageUuid,
     func: func,
     timestamp: Date.now(),
+    monotonicTimestamp: getCurrentTimestamp(),
```

Comment:
can you please confirm one-way messages from App to Hub also go through this function?

Reply:
Yes, I can confirm. Normally one-way app to host functions use `sendMessageToParent` which uses `sendMessageToParentHelper` which uses `createMessageRequest`. Here is an example with `notifySuccess`:
![image](https://github.com/user-attachments/assets/7c1c4afe-53e1-42bd-9324-0361668d68a9)